test/test_model.py

```diff
@@ -1469,5 +1469,56 @@ def test_bmp_sum(self):
         total = runoff + et +inf
         self.assertAlmostEqual(total, precip)
 
+    def test_BMPs_on_D(self):
```

Comment:
This is the only function that has capital letters. I'd change the name to follow the rest of the convention in the file.